test/__init__.py

```diff
@@ -374,33 +388,36 @@ def _init_client(self):
                 self.nodes = set([(host, port)])
             self.w = len(ismaster.get("hosts", [])) or 1
             self.version = Version.from_client(self.client)
-            self.server_parameters = self.client.admin.command(
-                'getParameter', '*')
 
-            if 'enableTestCommands=1' in self.cmd_line['argv']:
+            if TEST_SERVERLESS:
                 self.test_commands_enabled = True
-            elif 'parsed' in self.cmd_line:
-                params = self.cmd_line['parsed'].get('setParameter', [])
-                if 'enableTestCommands=1' in params:
+                self.has_ipv6 = False
```

Comment:
Serverless doesn't support ipv6?

Reply:
I am not sure... It seems we only use this in one test (`TestClient.test_ipv6`) - do we want to run that test class against serverless?

Reply:
Ah okay, I don't think we need to run that test on serverless. A maybe useful test would be to manually resolve the serverless host to ipv6 and then attempt to connect. Up to you. I don't think it's needed. Should be something that's tested on the cloud team's side.